pkg/cephmgr/osd/daemon.go

```diff
@@ -35,7 +37,7 @@ const (
 	dirOSDConfigFilename = "osd-dirs"
 )
 
-func Run(dcontext *clusterd.DaemonContext, agent *OsdAgent) error {
+func Run(dcontext *clusterd.DaemonContext, agent *OsdAgent, usingDeviceFilter bool) error {
```

Comment:
It could possible be internal to the agent, derived from which flag got set (`data-devices` or `data-device-filter`), i'll look into that.